maquetado/scss/login.scss

```diff
@@ -0,0 +1,94 @@
+@import "colors";
+
+body {
+  display: flex;
+  justify-content: center;
+}
+
+.wolox-image-login {
+  align-self: center;
+  height: 70px;
+  margin-top: 20px;
+  width: 200px;
+}
+
+.login-container {
+  align-items: center;
+  background-color: $light-grey;
+  border-top: 10px solid $light-blue;
+  display: flex;
+  flex-direction: column;
+  height: 400px;
+  justify-content: flex-start;
+  margin-top: 70px;
+  width: 300px;
+  max-width: 300px;
+}
+
+.label {
+  align-self: flex-start;
```

Comment:
Here it does have sense because the parent has everything set as align-items: center. Although why you set them like that and not everything to the left ?